### ISSUE TITLE
How to encode Ogg/vorbis from wav?

### ISSUE THREAD

**Issue by vitaly-zdanevich** [xlab/vorbis-go#6]
Looks like no documentation about that in README.md.

**Comment by xlab** [comment]
WAV is usually combined with uncompressed audio, but Ogg Vorbis is a general-purpose compressed audio format.

https://en.wikipedia.org/wiki/WAV
https://xiph.org/vorbis/

**Comment by vitaly-zdanevich** [comment]
Yes, description of this repository: "Package vorbis provides Go bindings for OggVorbis", so with this `vorbis-go` this is impossible to encode uncompressed wav to compressed Ogg/vorbis? Only option that I have is to call `oggenc` from Go through the `os/exec`?

**Comment by xlab** [comment]
Sorry I misread the title as "how to decode..." :)

This example https://github.com/xiph/vorbis/blob/master/examples/encoder_example.c can be ported to Go as I did with their decoder example. I just needed to decode, so I ported only one-way example.

**Comment by vitaly-zdanevich** [comment]
While this is not ported to your `vorbis-go` - Go's `os/exec` to `oggenc` is the only option?

**Comment by xlab** [comment]
I think so, or bindings to ffmpeg.

Feel free to take this issue to [https://gitcoin.co](https://gitcoin.co/bounty/quickstart)
